proc/threads.yaml

```diff
@@ -0,0 +1,7 @@
+test:
+    tests:
+        - name: test_threads_rand
```

Comment:
why `threads` / `thread`?

Reply:
My mistake. Changed plural form to singular `thread`